code/src/library/Containers.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components'
+
+export const Main = styled.main`
+${({noList}) => noList?`height: 100vh;`:`height:100%;`}
```

Comment:
I can't see that this work. Height is 100vh even though I have added todos to the list.